src/hooks/use-site-details.tsx

```diff
@@ -285,14 +285,27 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 			try {
 				updatedSite = await getIpcApi().startServer( id );
 			} catch ( error ) {
-				getIpcApi().showErrorMessageBox( {
-					title: __( 'Failed to start the site server' ),
-					message: __(
-						"Please verify your site's local path directory contains the standard WordPress installation files and try again. If this problem persists, please contact support."
-					),
-					error,
-					showOpenLogs: true,
-				} );
+				if (
+					error instanceof Error &&
+					error.message.includes( 'Studio failed to start the proxy server' )
```

Comment:
If this error only indicates that port 80 is busy, I would argue we should use a more specific error message.

Reply:
I've improved wording in e06d0622f27fa25b89763b09faf8ee43b379b019

Reply:
I agree we could use a message like `PROXY_ERROR_80`, since this is only used internally 👍